index.html

```diff
@@ -0,0 +1,463 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars">
+        <i
+          class="fas fa-bars hamburger-open-menu"
+          id="hamburger-close-menu"
+        ></i>
+      </button>
+      <nav id="desktop-nav">
+        <ul>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
+          </li>
+        </ul>
+      </nav>
+
+      <div class="mobile-menu-container">
+        <i
+          class="fas fa-times fa-2x hamburger-close-menu"
+          id="mobile-close-menu"
+        ></i>
+        <ul>
+          <li class="mobile-nav-link"><a href="#about-me">About</a></li>
+          <li class="mobile-nav-link"><a href="about-me">skills</a></li>
```

Comment:
- [ ] When the user clicks (or taps) the skill menu option, an incorrect part of the page is displayed. Please fix this issue, Correcting this type of bugs makes your portfolio looks professional.

Reply:
- [ ] Hi @petudeveloper,
      Thanks so much I will address the issue.